composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require-dev": {
         "wp-cli/scaffold-command": "^1.2 || ^2",
-        "wp-cli/wp-cli-tests": "^2.0.7"
+        "wp-cli/wp-cli-tests": "^2.0.14"
```

Comment:
```suggestion
        "wp-cli/wp-cli-tests": "^2.1"
```

Reply:
That was a large part of the reason that this is a draft PR :stuck_out_tongue_winking_eye: